.github/ISSUE_TEMPLATE/bug_report.yml

```
@@ -0,0 +1,211 @@
+
+        Please **fill out all the fields and checkboxes of this form** to make it easier for maintainers to understand the problem and to solve it. If this template is not adhered to, it will be marked with a label called `meta/no template - no support` and will not be worked on.
+
+        **Make sure** you read through the whole README, the documentation and the issue tracker before opening a new bug report.
+  - type: checkboxes
+    id: miscellaneous-first-checks
+    attributes:
+      label: Miscellaneous first checks
+      description: Please read these carefully and check the appropriate boxes.
+      options:
+        - label: I checked that all ports are open and not blocked by my ISP / hoster.
+          required: true
+        - label: I know that SSL errors are likely the result of a wrong setup on the user side and not caused by DMS itself. I'm confident my setup is correct.
+          required: true
+  - type: input
+    id: affected-components
+    attributes:
+      label: Affected Component(s)
+      description: What is affected by this bug? Please describe it with only a few words here. Detailed description can be given later.
+      placeholder: No debug output is shown.
+    validations:
+      required: true
+  - type: textarea
+    id: when-does-it-occur
+    attributes:
+      label:  When does this occur and what happened?
+      description: Tell us what happened. Use [fenced code blocks](https://docs.github.com/en/github/writing-on-github/working-with-advanced-formatting/creating-and-highlighting-code-blocks#fenced-code-blocks) when pasting lots of text!
+      placeholder: Although `DMS_DEBUG=1` is set, I see no debug output.
+      render: Markdown
+    validations:
+      required: true
+  - type: textarea
+    id: what-did-you-expect-to-happen
+    attributes:
+      label:  What did you expect to happen?
+      description: Tell us what you expected.
+      placeholder: I expected to see debug messages.
+      render: Markdown
+    validations:
+      required: true
+  - type: textarea
+    id: how-do-we-replicate-this-issue
+    attributes:
+      label: How do we replicate the issue?
+      description: What did you do and how can we replicate this issue?
+      value: |
+        1.
+        2.
+        3.
+        ...
+      render: Markdown
+    validations:
+      required: true
+  - type: input
+    id: mailserver-version
+    attributes:
+      label: DMS version
+      description: With which version did yor encounter this bug? Paste your image tag here.
+      placeholder: v10.1.2
+    validations:
+      required: true
+  - type: dropdown
+    id: main-memory
+    attributes:
+      label: How much RAM is available to DMS explicitly?
+      description: Please provide the amount of memory that DMS can really use. This includes main memory (RAM), but also swap or zram. If you're using Kubernetes, provide the resource limit here if you set one.
```

Does this imply some k8s support?

```suggestion
      description: Please provide the amount of memory that DMS can really use. This includes physical memory (RAM), but also virtual memory (swap or zram). If you're using Kubernetes, provide the resource limit here if you set one.
```